sdk-setup/src/mer-sdk-chroot

```diff
@@ -283,12 +284,18 @@ prepare_user() {
     install --owner $user --group $(id -gn $user) -d ${sdkroot}${HOMEDIR}
 
     if [[ $bind_mount_home == "yes" ]] ; then
-	echo "Mounting home directory: ${HOMEDIR}"
-	mount --bind ${HOMEDIR} ${sdkroot}${HOMEDIR}
-	# Now the sdk uses a private namespace, there's no need to
-	# make it unbindable
-	mount --make-shared ${sdkroot}${HOMEDIR}
+    	echo "Mounting home directory: ${HOMEDIR}"
+        mount --bind ${HOMEDIR} ${sdkroot}${HOMEDIR}
```

Comment:
Bad indent, avoid whitespace change (keep tabs in old code, use spaces in new code).